app/assets/stylesheets/application.css.scss

```diff
@@ -35,4 +35,17 @@
 
 .block_user_current_block_id {
   margin-left: 20px;
-}
\ No newline at end of file
+}
+
+#flickr_image a {
+  margin-bottom: 15px;
+}
+
+#flickr_images {
+  margin-bottom: 0;
+}
+
```

Comment:
Avoid using id selectors<br>Selector `load_flickr_image` should be written in lowercase with hyphens<br>Selector `load_local_image` should be written in lowercase with hyphens